src/components/znkSession/services/znkSession.service.js

```diff
@@ -0,0 +1,203 @@
+                    }
+                }
+                function minToUnixTimestamp(min) {
+                    return min * 60 * 1000;
+                }
+                function getPath(param) {
+                    if (!userAuth) {
+                        $log.error('Invalid user');
+                        return;
+                    }
+                    var path;
+                    var educatorUID = isTeacherApp ? userAuth.uid : TeacherContextSrv.getCurrUid();
+                    var studentUID = isTeacherApp ? StudentContextSrv.getCurrUid() : userAuth.uid;
+                    switch (param) {
+                        case 'sessions':
+                            path = ENV.studentAppName + '/liveSession/' + currLiveSessionsGUID.guid;
+                            return path;
+                        case 'student':
+                            path = ENV.studentAppName + '/users/$$uid/liveSession';
+                            return path.replace('$$uid', '' + studentUID);
+                        case 'educator':
+                            path = ENV.dashboardAppName + '/users/$$uid/liveSession';
+                            return path.replace('$$uid', '' + educatorUID);
+                        default:
+                            return;
+                    }
+                }
+                function saveSession() {
+                    $log.debug('saveSession, sessionData: ', sessionData);
+                    var dataToSave = {};
+                    globalStorageProm.then(function (globalStorage) {
+                        var studentPath = getPath('student') + '/active';
+                        var educatorPath = getPath('educator') + '/active';
+                        var sessionPath = getPath('sessions');
+                        dataToSave[sessionPath] = sessionData;
+                        dataToSave[studentPath] = { guid: sessionData.sessionGUID };
+                        dataToSave[educatorPath] = { guid: sessionData.sessionGUID };
+                        globalStorage.update(dataToSave);
+                    });
+                }
+                function updateSession() {
+                    $log.debug('updateSession, sessionData: ', sessionData);
+                    var dataToSave = {};
+                    globalStorageProm.then(function (globalStorage) {
+                        var studentPathActive = getPath('student') + '/active';
+                        var studentPathArchive = getPath('student') + '/archive/' + sessionData.sessionGUID;
+                        var educatorPathActive = getPath('educator') + '/active';
+                        var educatorPathArchive = getPath('educator') + '/archive/' + sessionData.sessionGUID;
+                        var sessionPath = getPath('sessions');
+                        dataToSave[sessionPath] = sessionData;
+                        dataToSave[studentPathArchive] = false;
+                        dataToSave[educatorPathArchive] = false;
```

Comment:
why not 

dataToSave[studentPathArchive] =dataToSave[educatorPathArchive] = false.
It will remove several lines of code and for me will be more readable